getting-started/data-scientists.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Data scientists
+description: Get started with Coder as a data scientist.
+---
+
+This article will walk you through the process of getting started with a Coder
+workspace capable of supporting data science projects. You'll learn how to:
+
+- Connect Coder to your Git provider;
+- Create a workspace with Jupyter and other data science packages present;
+- Add a sample project to your workspace, specifically
+  [one in Jupyter notebook form using IMDB movie data](https://github.com/khorne3/data-science-imdb-sample)
+- Create a dev URL and preview changes to your project.
+
+## Prerequisites
+
+This guide assumes that you have a Coder deployment available to you and that
+you have the credentials needed to access the deployment.
+
+## Step 1: Log in and connect Coder to your Git provider
+
+In this step, you'll log into Coder and connect and authenticate with your Git
+provider. This will allow you to do things like pull repositories and push
+changes.
+
+1. Navigate to the Coder deployment using the URL provided to you by your site
+   manager, and log in.
+
+1. Click on your avatar in the top-right, and select **Account**.
+
+   ![Set account preferences](../assets/getting-started/account-preferences.png)
+
+1. Provide Coder with your SSH key to connect and authenticate to GitHub.
+
+   If your site manager has configured OAuth, go to **Linked Accounts** and
+   follow the on-screen instructions to link your GitHub account.
+
+   ![Link GitHub account](../assets/getting-started/linked-accounts.png)
+
+   If your site manager has _not_ configured OAuth, go to **SSH keys**. Copy
```

Comment:
"or you are using a Git provider that Coder does not support" (this also applies if someone is using something other than GitHub, GitLab, Bitbucket, that we support)